src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -113,5 +118,11 @@
     <ProjectReference Include="..\..\System.Private.Uri\pkg\System.Private.Uri.pkgproj">
     </ProjectReference>
   </ItemGroup>
+  <!-- ToDo: Remove this target once the infrastructure for testing in netcoreapp1.1 is ready -->
+  <Target Name="RemoveUnwantedTestMonikers" BeforeTargets="CopyTestToTestDirectory">
```

Comment:
One small thing that I forgot, we should probably have a Condition for ns17 only in here that looks like:
```xml
<Target Name="RemoveUnwantedTestMonikers" Condition="'$(TargetGroup)'=='netstandard1.7'" BeforeTargets="CopyTestToTestDirectory">
```